src/sidebar/icons.js

```diff
@@ -1,3 +1,5 @@
+// @ts-nocheck - TS doesn't understand `require('.../icon.svg')`
```

Comment:
...why?, she asked. Because it's not using ES6 imports?

Reply:
We have a custom Browserify plugin that allows .svg files to be imported as if they were JS files, but TS doesn't know about that.